content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/go.md

```diff
@@ -89,23 +77,24 @@ To install and set up Orchestrion:
    go install github.com/DataDog/orchestrion@latest
    ```
    <div class="alert alert-info">Ensure <code>$(go env GOBIN)</code> (or <code>$(go env GOPATH)/bin</code>) is in your <code>$PATH</code>.</div>
-   
+
 1. Register Orchestrion in your project's `go.mod`:
    ```sh
    orchestrion pin
    ```
-1. Commit changes to your version control system:
+   Refer to the output of `orchestrion pin -help` for more information about available customization options.
+1. Commit changes to your version control system (unless you are integrating `orchestrion` directly in your CI/CD pipeline):
    ```sh
    git commit -m "chore: enable orchestrion" go.mod go.sum orchestrion.tool.go
    ```
 
-Now you can manage your dependency on `orchestrion` like any other dependency using the `go.mod` file.
+   Now you can manage your dependency on `orchestrion` like any other dependency using the `go.mod` file.
```

Comment:
I indented it into the list item because this is only accurate/relevant if you've checked in the changes (meaning you're not doing a "pure CI/CD integration").

That being said, the distinction is subtle and I wouldn't mind un-indenting that if you still think it's better.

Reply:
Ahh, I see what you mean... I think it's fine as-is then. :)